fast_api_als/utils/adf.py

```diff
@@ -4,7 +4,7 @@
 from uszipcode import SearchEngine
 import re
 
-
+logging.basicConfig(filename='adf.log', format="%(levelname)s: %(message)s", level=logging.DEBUG)
```

Comment:
instead of adding this config in each module, create a util for your custom logger and import wherever required